src/main/java/ch/njol/skript/expressions/ExprEnchantmentExpLevelCosts.java

```diff
@@ -0,0 +1,194 @@
+/**
+ *   This file is part of Skript.
+ *
+ *  Skript is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  Skript is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with Skript.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ *
+ * Copyright 2011-2017 Peter Güttinger and contributors
+ */
+package ch.njol.skript.expressions;
+
+import java.util.Arrays;
+
+import org.bukkit.enchantments.EnchantmentOffer;
+import org.bukkit.event.Event;
+import org.bukkit.event.enchantment.PrepareItemEnchantEvent;
+import org.eclipse.jdt.annotation.Nullable;
+
+import ch.njol.skript.ScriptLoader;
+import ch.njol.skript.Skript;
+import ch.njol.skript.classes.Changer;
+import ch.njol.skript.classes.Changer.ChangeMode;
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Events;
+import ch.njol.skript.doc.Examples;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.RequiredPlugins;
+import ch.njol.skript.doc.Since;
+import ch.njol.skript.lang.Expression;
+import ch.njol.skript.lang.ExpressionType;
+import ch.njol.skript.lang.SkriptParser.ParseResult;
+import ch.njol.skript.lang.util.SimpleExpression;
+import ch.njol.skript.log.ErrorQuality;
+import ch.njol.util.Kleenean;
+import ch.njol.util.coll.CollectionUtils;
+
+@Name("Enchant Prepare Enchantment Experience Level Cost")
+@Description({"The experience cost of an enchantment in an enchant prepare event.",
+			"If the cost is changed, it will always be at least 1."})
+@Examples({"on enchant prepare:",
+		"\tset the cost of enchantment 1 to 50"})
+@Since("INSERT VERSION")
+@Events("enchant prepare")
+@RequiredPlugins("1.9 or 1.10")
+@SuppressWarnings("deprecation")
+public class ExprEnchantmentExpLevelCosts extends SimpleExpression<Number>{
+
+	static {
+		if (!Skript.isRunningMinecraft(1, 11)) { // This expression should only be usable on 1.9 and 1.10.
+			Skript.registerExpression(ExprEnchantmentExpLevelCosts.class, Number.class, ExpressionType.SIMPLE,
+					"[the] cost of (enchant[ment]s|enchant[ment] offers)",
+					"[the] cost of enchant[ment] [offer] %number%",
```

Comment:
instead of a whole expression for this, couldn't you just have this just return an `EnchantmentOffer` and use the already existing cost expression

Reply:
Since EnchantmentOffer wasn't added until 1.11, this expression exists to give users on 1.9 / 1.10 some ability to make changes. The word "offer" in this expression is a little misleading.

Reply:
maybe you could create skript's own EnchantmentOffer and register that on old versions kind of like how ItemType wraps ItemStack and Color wraps Bukkit's color

Reply:
That could be interesting, although this is the only customizable part on 1.9 / 1.10 I believe.